tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.Integration.Infrastructure;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.Identity.Test.Unit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.net45.HeadlessTests
+{
+    [TestClass]
+    public class ConfidentialClientIntegrationTests
+    {
+        private static readonly string[] s_scopes = { "User.Read" };
```

Comment:
>s [](start = 41, length = 1)

why s_ / static? any reason these are not consts?

Reply:
For an array yes, you can't have const, static readonly is the closest it can get to const.

Reply:
I need to pay more attention to detail...

Reply:
aside: I was also asking for the naming

Reply:
I believe s_/ does refer to it being static. I followed the naming convention that I saw in other test classes.